reverse/reverse.py

```diff
@@ -38,10 +38,14 @@ def contains(self, value):
 
         return False
 
```

Comment:
Reverse list about changing the pointers.
think about the current **node** and **next_node** and how we reassign and refer to these nodes.

**Head** [next Node A] > **Node A** [next Node B]> **Node B** [next Tail]> **Tail** [next None] > None